pubsub/google/cloud/pubsub_v1/subscriber/_consumer.py

```diff
@@ -244,10 +244,9 @@ def _blocking_consume(self):
                 self.stop_consuming()
             except Exception as exc:
                 try:
-                    self._policy.on_exception(exc)
+                    self.active = self._policy.on_exception(exc)
                 except:
                     self.active = False
```

Comment:
Is this branch needed any longer?

Reply:
It depends. Do we want to prepare for the case that implementations of `on_exception` can fail?

Reply:
I think it's less useful to swallow the exception, as it would be truly exception if `on_exception` exceptioned.

Reply:
Agreed.